.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
This is debatable and I'm not opinionated.

Reply:
i agree with this value, we want people to use the structured issue. (they can always delete the text if they need a blank option)